pkgs/tools/security/zmap/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, libjson, json_c, gengetopt, flex, byacc, gmp
+, libpcap
+}:
+
+stdenv.mkDerivation rec {
+  pname = "zmap";
+  version = "2.1.1";
+
+  src = fetchFromGitHub {
+    owner = "zmap";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0yaahaiawkjk020hvsb8pndbrk8k10wxkfba1irp12a4sj6rywcs";
+  };
+
+  patches = [ ./0001-The-byacc-binary-in-nixpkgs.byacc-is-named-yacc.patch ];
+
+  cmakeFlags = [ "-DRESPECT_INSTALL_PREFIX_CONFIG=ON" ];
```

Comment:
So this will install the config within our prefix. Will it still read it at `/etc`? This is a common problem.

Reply:
It seems as it still reads from `/etc/zmap/` by default, but it's changeable using `-C`.

> This is a common problem.


I know. This will work fine when installing the package in `environment.systemPackages`, but will cause problems if not I guess. What would you suggest here?